CHANGELOG.md

```diff
@@ -11,7 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
    Unreleased section, uncommenting the header as necessary.
 -->
 
-<!--## Unreleased-->
+## Unreleased
+
+* Fix an issue where package-external features could be included in queries that explicitly asked for them not to be. Specifically imports like `../paper-button/paper-button.html` weren't considering the resolved path like `bower_components/paper-button/paper-button.html` when determining externality.
```

Comment:
not major, but a little confused by what you mean that "imports wren't considering their own resolved path". Shouldn't this be that the analyzer isn't considering resolvedness?